code/src/MovieList.css

```diff
@@ -0,0 +1,114 @@
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+    monospace;
+}
+
+.allMovies {
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  min-width: 375px;
+}
+
+.eachMovie {
+  display: block;
+  word-wrap: break-word;
+  position: relative;
+  width: 187px;
+  height: 254px;
+}
+
+.movieThumbnail {
+  width: 100%;
+  height: 100%;
+}
+
+.movieTitle1 {
+  display: none;
+}
```

Comment:
Very clearly structured CSS clasess from top to the bottom of the page. Makes it so easy to follow. 😍